moninag/static/src/ts/app/servers/server-add.component.html

```diff
@@ -0,0 +1,20 @@
+<div>
+    <form [formGroup]="serverForm" (ngSubmit)="submitForm(serverForm.value)" >
+    <div class="form-group">
+        <input class="form-control" type="text" placeholder="Server Name" [formControl]="serverForm.controls['name']">
+    </div>
+    <div class="form-group">
+        <input class="form-control" type="text" placeholder="Server Address" [formControl]="serverForm.controls['address']">
+    </div>
+    <div class="form-array">
+        <label>State:
+        <select class="form-control" formControlName="state">
+        <option *ngFor="let state of states" [value]="state">{{state}}</option>
+        </select>
+        </label>
```

Comment:
`label` is inline tag, so it would be inline. The reason is that you put in label not only label, but full select block.
Again correct indents :slightly_smiling_face: